components/brave_vpn/browser/connection/ikev2/mac/brave_vpn_ras_connection_api_mac.mm

```diff
@@ -62,21 +62,6 @@
   return (__bridge NSData*)copy_result;
 }
 
-OSStatus RemoveKeychainItemForAccount() {
```

Comment:
Just to confirm: macOS is removing the entry when you uninstall the app?

Reply:
no, uninstallation on MacOS is just removing the Brave.app folder, there is no code execution from our side

Reply:
Ah - that's right. I think the VPN config is partitioned by application. If you empty the trash can, I'm curious if networking removes the entry for Brave already?